resources/views/graph.blade.php

```diff
@@ -43,79 +72,76 @@
 <div class="container">
   <canvas id="graph"></canvas>
    <br/><br/><br/>
+   @if(!empty($decodeRes['result']))
   <div class"row justify-content-center">
-    <table width="100%">
+    <table width="100%" style="margin-left:auto; margin-right:auto;">
       <tr>
-        <th width="5%"></th>
-        <th width="5%"></th>
-        <th width="5%"></th>
-        <th width="5%"></th>
-        <th width="5%"></th>
-        <th width="5%"></th>
-        <th width="5%"></th>
-        <th width="5%"></th>
-        <th width="5%"></th>
-        <th width="5%"></th>
+        <th width="4%"></th>
+        <th width="4%"></th>
+        <th width="4%"></th>
+        <th width="4%"></th>
+        <th width="4%"></th>
+        <th width="4%"></th>
+        <th width="4%"></th>
+        <th width="4%"></th>
+        <th width="4%"></th>
+        <th width="4%"></th>
+        <th width="4%"></th>
+        <th width="4%"></th>
+        <th width="4%"></th>
+        <th width="4%"></th>
+        <th width="4%"></th>
+        <th width="4%"></th>
+        <th width="4%"></th>
+        <th width="4%"></th>
+        <th width="4%"></th>
+        <th width="4%"></th>
+        <th width="4%"></th>
+        <th width="4%"></th>
+        <th width="4%"></th>
+        <th width="4%"></th>
+        <th width="4%"></th>
       </tr>
       <tr>
         <?php
-        $numSad = $result/10;
-        for($j = 0; $j <1; $j++){
-          echo "<tr>";
-          for($i = 0; $i < 10; $i++ ){
-            if($numSad > 0){
-          ?>
-              <td><img src="{{ URL::to('/images/highRisk.png') }}" width="100%" /></td>
-          <?php
-              $numSad = $numSad -1;
-            }
-            else{
-          ?>
-            <td><img src="{{ URL::to('/images/lowRisk.png') }}" width="100%"/></td>
-          <?php
+          $numSad = $decodeRes["result"];
+          for($j = 0; $j <4; $j++){
```

Comment:
use blade template syntax (`@for`, `{{ }}`, `@if` etc.) here and in subsequent lines